x/emissions/keeper/inference_synthesis/network_losses.go

```diff
@@ -112,6 +112,15 @@ func CalcNetworkLosses(
 			if err != nil {
 				return emissions.ValueBundle{}, err
 			}
+			oneE18, err := alloraMath.NewDecFromString("1000000000000000000")
```

Comment:
should be a constant that gets imported from somewhere so you don't have to redefine each time